ui/intermittent-failures/BugDetailsView.jsx

```diff
@@ -141,6 +142,10 @@ const BugDetailsView = props => {
       header={
         <React.Fragment>
           <Row>
+            <Helmet>
+              <meta charSet="utf-8" />
+              <title>{summary}</title>
```

Comment:
I've tested this and it's certainly an improvement but it only contains partial information; the bug number, so `bug 12345 - whatever the summary is` should also be included. 

Also, I don't think you answered my question about the meta tag and why that's being used here. :)

Reply:
The meta tag is created uniquely for that page, in this case, for the particular bug page.Interestingly, I googled that the meta tag won't be duplicated but rather it'll override so for an example, if a user was to be on the particular bug page, the meta tag will be unique to that page rather than the main index page.
A question I have is, do I create a react-helmet on the index and then create parent and children relationship with meta tags?

Reply:
Also, I tried to replace the tab to show `bug 12345 - whatever the summary is` with 
            `<Helmet>
              <title>Bug {bug} - {summary}</title>
            </Helmet>`
and it wasn't showing up for me. Am I missing something?

Reply:
> Also, I tried to replace the tab to show `bug 12345 - whatever the summary is` with
> `<Helmet> <title>Bug {bug} - {summary}</title> </Helmet>`
> and it wasn't showing up for me. Am I missing something?

You need to use a template literal.

Reply:
> The meta tag is created uniquely for that page, in this case, for the particular bug page.Interestingly, I googled that the meta tag won't be duplicated but rather it'll override so for an example, if a user was to be on the particular bug page, the meta tag will be unique to that page rather than the main index page.
> A question I have is, do I create a react-helmet on the index and then create parent and children relationship with meta tags?

I don't know what the benefit of that would be - the docs aren't very clear. We only want to change the title for bug details view so I think this is fine.

Reply:
Sorry, only because I’ve been trying different things out. I did and go through with using a back ticks, dollar sign and curly braces for expressions. On the tab, it shows as it is. How I initially wrote the expression in the curly braces worked. Previously having ‘{bug}’ and ‘{summary}’ worked but then when I refreshed the page, it went blank. I’m hoping to figure it out a bit more tonight but if not, I would like to know what you think. I asked the public chat if anyone knew react helmet but I didn’t get an answer so I’m thinking it fairly new everyone. 

Other than adding the bug detail number, I’m really thankful and appreciative to have you work so patiently.

Reply:
If a page goes blank, then you'll need to open the devtools panel and look in the console for errors. This is an important tool for development so it's a good thing to get familiar with :) Also, frequently running `yarn lint --fix` in your terminal will catch errors that could be causing the page to break (go blank).

For the title, what you'd need to do is this: `{`Bug ${bug} - ${summary}`}`. Having to wrap a template literal/string in curly brackets is an aspect of using jsx in React.